lms/validation/__init__.py

```diff
@@ -58,14 +54,6 @@ def foo_view(request):
 )
 
 
-parser = pyramidparser.PyramidParser()
-
-
-@parser.error_handler
-def _handle_error(error, _req, _schema, _status_code, _headers):
-    raise ValidationError(messages=error.messages) from error
```

Comment:
All this `parser` and `_handle_error()` stuff has been moved into `PyramidRequestSchema` which is the only place where it's needed, since it's specific to validating Pyramid requests